code/condition_variable/condition_variable.cpp

```diff
@@ -61,7 +61,7 @@ bool process(unsigned int threadIdx, Data const & data) {
 
   // Burn some CPU cycles to simulate intensive data processing.
   const auto startTime = std::chrono::high_resolution_clock::now();
-  int dummyCounter = 0;
+  unsigned dummyCounter = 0;
   while (std::chrono::high_resolution_clock::now() - startTime < 5s) {
     ++dummyCounter;
   }
```

Comment:
Needs to be unsigned, because it can intentionally overflow and should wrap around. Signed int overflow is UB.